VSConfigFinder/CommandLine/VSConfig.cs

```diff
@@ -10,7 +10,7 @@ namespace VSConfigFinder
     /// <summary>
     /// The class object that defines a .vsconfig file.
     /// </summary>
-    internal class VSConfig
+    public class VSConfig
     {
         /// <summary>
         /// Gets or sets the version of the .vsconfig file.
```

Comment:
When I deserialize json to read each vsconfig, the VSConfig class parameter and json object keys had to match (case sensitive). I think we have to change "Version" to "version" and "Components" to "components".

Reply:
@jlee671 Have you been using properties to ignore case sensitivity when creating `JsonSerializerOptions`? You shouldn't need to care about that if you set `PropertyNameCaseInsensitive = true`.